src/app/features/home/details/details.page.ts

```diff
@@ -208,35 +199,6 @@ export class DetailsPage {
 
   userToken: string | undefined;
 
-  readonly iframeUrlWithToken$ = combineLatest([
-    this.activeDetailedCapture$,
-  ]).pipe(
-    distinctUntilChanged(),
-    map(([detailedCapture]) => {
-      const token = this.userToken;
-      const params = `nid=${detailedCapture.id}&token=${token}&from=mycapture`;
-      const url = `${BUBBLE_IFRAME_URL}/asset_page?${params}`;
-      return this.sanitizer.bypassSecurityTrustResourceUrl(url);
-    })
-  );
-
-  readonly iframeUrlWithJWTToken$ = combineLatest([
-    this.activeDetailedCapture$,
-    this.diaBackendAuthService.cachedQueryJWTToken$,
-    this.iframeService.detailsPageIframeReloadRequested$,
-  ]).pipe(
-    distinctUntilChanged(),
-    map(([detailedCapture, token, _]) => {
-      const params =
-        `nid=${detailedCapture.id}` +
-        `&token=${token.access}` +
-        `&refresh_token=${token.refresh}` +
-        `&from=mycapture`;
-      const url = `${BUBBLE_IFRAME_URL}/asset_page?${params}`;
-      return this.sanitizer.bypassSecurityTrustResourceUrl(url);
-    })
-  );
-
   readonly isFromSeriesPage$ = this.type$.pipe(map(type => type === 'series'));
 
   readonly networkConnected$ = this.networkService.connected$;
```

Comment:
ChatGPT Plus code review with improvement suggestions:

1.  Remove unused code: It seems that the `iframeUrl$`, `iframeUrlWithToken$`, and `iframeUrlWithJWTToken$` have been removed in this commit. Ensure that these variables were not used elsewhere in the `details.page.ts` file, and if they are, update the references to use the new implementation.

2.  Combine `combineLatest` operators: You have a single array passed to `combineLatest` for `activeDetailedCapture$`. You can simplify this by removing the `combineLatest` and using `activeDetailedCapture$` directly.

```typescript

// Replace this
readonly iframeUrlWithToken$ = combineLatest([
  this.activeDetailedCapture$,
]).pipe(
  // ...
);

// With this
readonly iframeUrlWithToken$ = this.activeDetailedCapture$.pipe(
  // ...
);```

1.  Ensure proper functionality: The code provided in this commit seems to remove a significant amount of logic related to iframe URLs. Double-check that the new implementation handles all required cases and the removal of the previous logic doesn't break any existing functionality.

2.  Formatting: Consider formatting the code consistently. For example, you have a line break before the `combineLatest` operator in the `iframeUrlWithToken$` definition, but no line break before the `distinctUntilChanged()` operator. Consistent formatting will make the code easier to read and maintain.

Other than these suggestions, the provided code seems to be okay. Make sure to test the changes thoroughly to ensure the application still functions as expected.